sprocs/sync_assessments.sql

```diff
@@ -166,13 +166,24 @@ BEGIN
         new_assessment_ids = array_append(new_assessment_ids, new_assessment_id);
 
         -- if it is a group work try to insert a group_config
+        -- update the size and permission
         IF (valid_assessment.data->>'group_work')::boolean THEN
             INSERT INTO group_configs
                 (course_instance_id,
-                assessment_id
+                assessment_id,
+                maximum,
+                minimum,
+                student_authz_create,
+                student_authz_join,
+                student_authz_leave
             ) VALUES (
                 syncing_course_instance_id,
-                new_assessment_id
+                new_assessment_id,
+                (valid_assessment.data->>'group_max_size')::bigint,
+                (valid_assessment.data->>'group_min_size')::bigint,
+                (valid_assessment.data->>'student_group_create')::boolean,
+                (valid_assessment.data->>'student_group_join')::boolean,
+                (valid_assessment.data->>'student_group_leave')::boolean
```

Comment:
Can we change this back to always sync from the JSON files to the DB? We want the DB to always be exactly the same as what is in the JSON.